shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -424,19 +474,21 @@ public Options getOptions() {
     outputFileOpt = new Option("o", "output", true, "local file to write the scan output to");
     namespaceOpt = new Option(ShellOptions.namespaceOption, "namespace", true,
         "namespace to display/set/delete properties for");
-
+    propFileOpt = new Option("p", "propFileOpt", true, "file containing properties to set");
```

Comment:
```suggestion
    propFileOpt = new Option("p", "propFile", true, "file containing properties to set");
```
Only the variable name needed to change. The full flag the user passes in should remain `propFile`.